x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/feature_tour/translations.ts

```diff
@@ -13,3 +13,18 @@ export const TOUR_TITLE = i18n.translate(
     defaultMessage: "What's new",
   }
 );
+
+export const SEARCH_CAPABILITIES_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.featureTour.8.2.searchCapabilities.title',
```

Comment:
nit: [i18n message types](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md#naming-convention) are intended to be appended as camelCase I believe